src/backend/assets/js/session.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+  var sideslider = $('[data-toggle=collapse-side]');
+  var sel = sideslider.attr('data-target');
+  var top = $('header[role=banner]').outerHeight();
+  var downButton = $('#down-button');
+  var sessionElem = $('.single-session');
+  var sessionId = Number(sessionElem.attr('id'));
+  var maxNoOfSessions = 1000;
```

Comment:
Is that mean only 1000 sessions can be bookmarked? As, it is hard-coding the value in the localStorage.
Can we calculate length of the session object to set Maximum sessions?
It will save the memory and also we will not be going with a random value ?